packages/variable-resolver/src/browser/variable.spec.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (C) 2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import * as chai from 'chai';
+import { Container, ContainerModule } from 'inversify';
+import { ILogger } from '@theia/core/lib/common';
+import { MockLogger } from '@theia/core/lib/common/test/mock-logger';
+import { Variable, VariableRegistry } from './variable';
+import { Disposable } from 'vscode-jsonrpc';
```

Comment:
I do not know whether it was intentional or not, but we usually user `@theia/core/common/disposable`. I guess the tooling did it.

Reply:
Yes, it was VSCode's work ))
Fixed import.